lib/parser.js

```diff
@@ -30,13 +31,16 @@ module.exports = {
  * @param {object} opts The options object
  * @param {string} [opts.config] The POJO of the config you want to validate.
  * @param {string} [opts.configPath] The path to the config on disk to validate.
+ * @param {string} [opts.basePath] The basepath is used to correctly resolve
```

Comment:
Would be good to add a blurb here about requiring basepath if you pass in a POJO.